exporters/otlp/otlptrace/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenTelemetry-Go OTLP Span Exporter
+
+OpenTelemetry Protocol (OTLP) Span Exporter.
+
+To constructs a new Otlptrace Exporter and starts it:
+
+```
+exp, err := otlptrace.New(ctx, opts...)
+```
```

Comment:
1. This snippet does not really help
1.  `otlptrace.New` does not accept `opts`. It accepts `client` which factory functions accepts options

I would rather prefer having an example similar to this one https://github.com/Submarinee/opentelemetry-go/blob/main/exporters/stdout/stdouttrace/example_test.go and put a hyperlink to it in README.

As for me, you can also consider simply removing this section. No information is better than misleading information 😉

Reply:
Can I understand your suggestion as the following ?
1. Delete line 5-9.
2. In the otlptracegrpc directory, create example_test.go file containing an example for  creating otlptracegrpc (similar to this one https://github.com/Submarinee/opentelemetry-go/blob/main/exporters/stdout/stdouttrace/example_test.go) 
3. Put  the hyperlink to the example_test.go fil in README

Reply:
Yes 👍

Reply:
I have already created the example_test.go file ,maybe it is not correct.
If it contains some mistakes ,hope you can help me modify it.
thank u~

Reply:
I have found out that we have an existing example here: https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector. I was not aware of it - I have found it when I was reviewing your code. I think it would be enough to reference it instead of creating yet another example.

Also please take a look at: https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#documentation and update the docs accordingly.

Reply:
Can I understand your suggestion as the following ?
1.delete the example_test.go file I wrote several days ago
2.Change the hyperlink to  [https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector](https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector)

Reply:
But I still wonder why [ https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector]( https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector) can be an appropriate example in otlptrace exporter document.
I regard it as a good example of collector rather than a good example of exporter.
Can you explain it to me?

Reply:
After looking again I think that you are correct that the https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector may be too complex. 

I am OK with keeping `example_test.go`. Still, I think it is worth adding a hyperlink to https://github.com/open-telemetry/opentelemetry-go/tree/main/example/otel-collector in the `README.md`.